process-engine/process_engine_runtime
# ♻️ Make ExternalTaskWorker immutable

## Changes

1. Make ExternalTaskWorker immutable
2. Replace the test ExternalTaskWorker with an actual ExternalTaskWorker
3. Update pg, sequelize and sqlite to fix vulnerability issues reported by `npm audit` (See attached log)
![Bildschirmfoto 2019-09-18 um 10 04 02](https://user-images.githubusercontent.com/15343316/65129487-a8970900-d9fb-11e9-8c56-67cc54e8b51e.png)

## Issues

Closes #410

PR: #412

## How to test the changes

The changes only affect the integration tests, since the clients are not an actual part of the runtime.
Run the tests, to see if everything still works as before.

To see how the actual implementations work, use the clients to make requests against the ProcessEngine.
